libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/initializeDashboardHandler/index.ts

```diff
@@ -187,14 +217,17 @@ function* loadExistingDashboard(
         PromiseFnReturnType<typeof loadLegacyDashboards>,
         PromiseFnReturnType<typeof loadDashboardPermissions>,
         PromiseFnReturnType<typeof loadDateHierarchyTemplates>,
+        PromiseFnReturnType<typeof loadFilterViews>,
         PromiseFnReturnType<typeof loadAccessibleDashboardList>,
     ] = yield all(calls);
 
     const {
-        dashboard,
+        dashboard: loadedDashboard,
         references: { insights },
     } = dashboardWithReferences;
 
+    const dashboard = applyDefaultFilterView(loadedDashboard, filterViews, config);
```

Comment:
is not this code executed also during target dashboard load when drill to dashboard?